meta-facebook/mfz-wc/src/platform/plat_hook.c

```diff
@@ -24,6 +24,18 @@ mp5990_init_arg mp5990_init_args[] = {
 	[1] = { .is_init = false, .iout_cal_gain = 0x01BF, .iout_oc_fault_limit = 0x0046 }
 };
 
+ina230_init_arg ina230_init_args[] = {
```

Comment:
format?

Reply:
The format should be OK because the run-clang-format is passed.
What is the consideration of this format?